zoom-meeting-sdk-linux/h/meeting_service_components/meeting_waiting_room_interface.h

```
@@ -1,29 +1,29 @@
-/*!
-* \file meeting_waiting_room_interface.h
-* \brief Meeting Service Waiting Room Interface.
-* Valid only for Zoom style user interface mode.
-*/
+/**
+ * @file meeting_waiting_room_interface.h
+ * @brief Meeting Service Waiting Room Interface.
+ * Valid only for Zoom style user interface mode.
+ */
 #ifndef _MEETING_WaitingRoom_INTERFACE_H_
 #define _MEETING_WaitingRoom_INTERFACE_H_
```

Clang (14.0.6)

[warning] declaration uses identifier '_MEETING_WaitingRoom_INTERFACE_H_', which is a reserved identifier

(bugprone-reserved-identifier)